packages/app/src/debug/empty/DebugEmptyView.vue

```diff
@@ -17,43 +17,130 @@
       </div>
       <slot name="cta" />
     </div>
-    <div class="flex flex-col my-40px w-full items-center">
-      <DebugTestLoadingContainer
-        width-class="w-[75%]"
-        dot-class="icon-light-gray-200"
-        :rows="loadingRows"
-      >
-        <template #header>
-          <div class="flex">
-            <div class="bg-white border rounded-md flex border-gray-100 py-4px px-8px text-14px text-gray-700 gap-8px items-center">
-              <i-cy-status-failed_x12 class="h-12px w-12px" />
-              <span>-</span>
-              <div
-                v-if="exampleTestName"
-                class="border-l border-gray-100 pl-8px"
-              >
-                {{ exampleTestName }}
+    <Slideshow
+      v-if="step !== undefined && steps"
+      v-model="step"
+      class="my-40px w-full"
+      :total-steps="steps.length"
+    >
+      <template #default="slotProps">
+        <DebugSlide
+          v-if="step < steps.length"
+          v-bind="slotProps"
+          :img="steps[step].img"
+          :title="steps[step].title"
+          :description="steps[step].description"
+        />
+        <div
+          v-else
+          class="flex flex-col w-full items-center"
+          data-cy="debug-default-empty-state"
+        >
+          <DebugTestLoadingContainer
+            width-class="w-full"
+            dot-class="icon-light-gray-200"
+            :rows="loadingRows"
+          >
+            <template #header>
+              <div class="flex justify-between">
+                <div class="bg-white border rounded-md flex border-gray-100 py-4px px-8px text-14px text-gray-700 gap-8px items-center">
+                  <i-cy-status-failed_x12 class="h-12px w-12px" />
+                  <span>-</span>
+                  <div
+                    v-if="exampleTestName"
+                    class="border-l border-gray-100 pl-8px"
+                  >
+                    {{ exampleTestName }}
+                  </div>
+                </div>
+                <!-- TODO: Make a PR to Design System for this icon -->
```

Comment:
This has been merged

Reply:
Updated in dc9971caf79e9b1e95b804571a02401b11bcaa80